### PR TITLE
[DOCS] Community friendly documentation

### PR DESCRIPTION
**Pull request by rphang**
## Description

There was still alot of confusions about workflows and ways that this repo was working on. Added documentation should help newcomers get the basics of tools we use and the **Rebase & Merge** workflow.

## Added

- chore: Apache license ([d2b1a2a](https://github.com/Hive-Five-project/hive-five/pull/24/commits/d2b1a2aa3455a039f2b3dbcd5351b812f75fc365))
- doc: Architecture & Git workflow ([e934176](https://github.com/Hive-Five-project/hive-five/pull/24/commits/e9341766b4283e6b8edbe1667a9fac91339341ae))
- doc: PR template ([5b10539](https://github.com/Hive-Five-project/hive-five/pull/24/commits/5b105393ae8b787b03a8882b26fc7d0cf1209d69))

## Changed

- doc: new README ([8b337a0](https://github.com/Hive-Five-project/hive-five/pull/24/commits/8b337a0004ea2dba34965407f0edecc8978d01ab))

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,97 +1,174 @@
...
+
+### 📈 Data analysis
+
+(Coming soon)
+
+### 📱 Mobile-friendly
+
+**HiveFive** is mobile-friendly. You can use it on your phone, tablet, or computer. The interface is responsive and adapts to your device.
+
+[![HiveFive - Mobile-friendly](docs/resources/mobile_friendly.png)](https://hivefive.online)
```
Comment (LedruRomane) [inline README.md]: Capture d’écran 2024-06-17 à 16 41 23

is normal ?

File: back/docs/README.md
Comment (LedruRomane) [inline back/docs/README.md:1]: J'mets de la doc ici ?

**Review by LedruRomane — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (rphang) [reply README.md]: I forgot! :(

File: back/docs/README.md
Comment (rphang) [reply back/docs/README.md:1]: As you wish !

**Pull request merged**